experimental/net/tcpconfig.go

```diff
@@ -0,0 +1 @@
+package net
```

Comment:
please simplify into one file, like net.go

Reply:
but we're still keeping the subpackage right?